code/game/objects/structures/mannequin.dm

```diff
@@ -109,6 +111,10 @@
 	var/datum/sprite_accessory/socks/socks = GLOB.socks_list[socks_name]
 	if(socks)
 		. += mutable_appearance(socks.icon, socks.icon_state, -BODY_LAYER)
+	var/datum/sprite_accessory/bra/bra = GLOB.bra_list[bra_name]
+	if(bra)
+		. += mutable_appearance(bra.icon, bra.icon_state, -BODY_LAYER)
```

Comment:
Add Skyrat edit comments.

I'm not going to do it everywhere because honestly this is a lot of work suggesting it everywhere, please do it yourself.